docs/other-api/adapter.md

```diff
@@ -139,7 +139,7 @@ import * as build from "../build";
 const handleRequest = createRequestHandler({ build });
 
 const handleEvent = async (event: FetchEvent) => {
-  const response = await handleAsset(event, build);
+  let response = await handleAsset(event, build);
```

Comment:
Kent changed this to const today

Reply:
This one should still be `let`, it gets reassigned on line 145

Reply:
Ahh ok, ignore the comment then